frontend/src/views/msg/unread.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <layout-content v-loading="$store.getters.loadingMap[$store.getters.currentPath]">
+
+    <el-radio-group v-model="selectType" style="margin-bottom: 15px;" @change="typeChange">
+      <el-radio-button v-for="(item,index) in msgTypes" :key="index" class="de-msg-radio-class" :label="item.value">{{ item.label }}</el-radio-button>
+
+    </el-radio-group>
+    <complex-table
+      :data="data"
+      :columns="columns"
+      :pagination-config="paginationConfig"
+      @select="select"
+      @search="search"
+    >
+
+      <el-table-column prop="content" :label="$t('commons.name')">
+        <template v-slot:default="scope">
```

Comment:
Codacy has a fix for the issue: [Expected '#default' instead of 'v-slot:default'.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7696927)
```suggestion
        <template #default="scope">
```